src/common/types/value.cpp

```diff
@@ -37,6 +37,14 @@ Value::Value(int64_t val) : type_(LogicalType::BIGINT), is_null(false) {
 	value_.bigint = val;
 }
 
+Value::Value(uint32_t val) : type_(LogicalType::UINTEGER), is_null(false) {
```

Comment:
These were here before and I removed them because it can be quite unclear which constructor is being called when there are many integer constructors (we had some problems with this before, particularly in e.g. initializer lists). Prefer using either Value::CreateValue<T> or Value::UINTEGER.